vagabond/gui/elements/Modal.cpp

```diff
@@ -84,6 +84,33 @@ void Modal::addTwoButtons(std::string left, std::string ltag,
 
 }
 
+void Modal::addThreeButtons(std::string left, std::string ltag,
+						  std::string center, std::string ctag,
+                          std::string right, std::string rtag)
+{
+	{
```

Comment:
Lines 92-95 and lines 106-109 are the same as the code in `addTwoButtons`, and only lines 99-102 are new. Replace lines 92-95 and 106-109 with a call to `addTwoButtons(left, ltag, right, rtag)` and keep the lines 99-102 in order to re-use existing code :)